ui-admin/src/study/participants/enrolleeView/ParticipantNotesView.tsx

```diff
@@ -9,7 +9,7 @@ import {
   ParticipantTask
 } from '@juniper/ui-core'
 import { ParticipantNoteModal } from './ParticipantNoteModal'
-import { renderEmptyMessage } from 'util/tableUtils'
+import { renderEmptyMessage } from '../../../util/table/tableUtils'
```

Comment:
looks like a bunch of these imports got rewritten as part of the refactor.

definitely not a blocker- i'll leave it up to you if you want to clean them up